wallet/createtx.go

```diff
@@ -1,125 +1,88 @@
-// Copyright (c) 2013-2015 The btcsuite developers
+// Copyright (c) 2013-2016 The btcsuite developers
 // Use of this source code is governed by an ISC
 // license that can be found in the LICENSE file.
 
 package wallet
 
 import (
-	"errors"
 	"fmt"
-	badrand "math/rand"
 	"sort"
-	"time"
 
 	"github.com/btcsuite/btcd/blockchain"
-	"github.com/btcsuite/btcd/chaincfg"
+	"github.com/btcsuite/btcd/btcec"
 	"github.com/btcsuite/btcd/txscript"
 	"github.com/btcsuite/btcd/wire"
 	"github.com/btcsuite/btcutil"
 	"github.com/btcsuite/btcwallet/waddrmgr"
+	"github.com/btcsuite/btcwallet/wallet/txauthor"
 	"github.com/btcsuite/btcwallet/wtxmgr"
 )
 
-const (
```

Comment:
I realize these constants aren't used any more, but comments like this have proved very helpful in the past (writing decred tests and the like).
Just wondering if this stuff is reproduced elsewhere or if maybe we could save it in documentation somewhere.  Not really sure.